### ISSUE TITLE
[BUG] Black screen in menu and access violation crash on loading any map

### ISSUE THREAD

**Issue by TheSeeOHZee** [hh79/gzdoomvr#92]
### GZDoom version

GZDoomVR 4.11.3

### Which game are you running with GZDoom?

Doom 2

### What Operating System are you using?

Windows 11

### Please describe your specific OS version

Windows 11 Home

### Relevant hardware info

Intel Core i7-13700F, NVIDIA GeForce RTX 3070, 32GB DDR5 Memory

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

This happens every single time.

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Start the game and it results in a black screen
2. Press Enter a couple times to load in a map
3. Game crashes upon starting the map

### Your configuration

```ini
https://gist.github.com/TheSeeOHZee/7db6c7ba80e23131e210dc1837963012
```

### Provide a Log

https://gist.github.com/TheSeeOHZee/470cb33c5407d8e434e6c628051ea24f

**Comment by TheSeeOHZee** [comment]
Request to mark this as closed as by default GZDoom 4.11.3 uses Vulkan which VR doesn't work with. Set vid_preferbackend to 0 to use OpenGL in the config file to get this working.

**Comment by hh79** [comment]
Yes, only OpenGL backend is supported.
